### ISSUE TITLE
Plotting gene trends doesn't work with only 1 lineage

### ISSUE THREAD

**Issue by rorymaizels** [theislab/cellrank#510]
If I fix the number of terminal states to 1, cr.pl.gene_trends() does not work. The lineage name is fixed to 'Lineage 0' which does not match the name of the terminal state inferred (in the example below, 'Beta'). 

While CellRank is useful for cases where there are multiple terminal states, it would still be good to be able to model the gene expression changes followed by a single lineage on its own (in my own analysis, I am looking at the trajectories of >5 cell types, some of which bifurcate and some of which do not, and so the ability to plot the gene trends of both bifurcating and linear trajectories with CellRank would be highly desirable).

```python
import scvelo as scv
import scanpy as sc
import cellrank as cr
import numpy as np

adata = cr.datasets.pancreas()

scv.pp.filter_and_normalize(adata, min_shared_counts=20, n_top_genes=2000)
sc.tl.pca(adata)
sc.pp.neighbors(adata, n_pcs=30, n_neighbors=30)
scv.pp.moments(adata, n_pcs=30, n_neighbors=30)

scv.tl.recover_dynamics(adata)
scv.tl.velocity(adata, mode='dynamical')
scv.tl.velocity_graph(adata)

cr.tl.terminal_states(adata, cluster_key='clusters', weight_connectivities=0.2, n_states=1)
cr.tl.initial_states(adata, cluster_key='clusters')
cr.tl.lineages(adata)
scv.tl.recover_latent_time(adata, root_key='initial_states_probs', end_key='terminal_states_probs')
cr.tl.lineage_drivers(adata)

model = cr.ul.models.GAM(adata, n_knots=6, spline_order=3)
cr.pl.gene_trends(adata, model, genes=adata.var_names[10])
```

The result:

![Screenshot 2021-03-01 at 16 29 24](https://user-images.githubusercontent.com/29228970/109527918-17f7a000-7aac-11eb-9597-adad0af9ad55.png)

#### Versions:

> cellrank==1.2.0 scanpy==1.6.0 anndata==0.7.5 numpy==1.19.4 numba==0.52.0 scipy==1.5.4 pandas==0.25.3 scikit-learn==0.23.2 statsmodels==0.12.1 python-igraph==0.8.3 scvelo==0.2.2 pygam==0.8.0 matplotlib==3.3.3 seaborn==0.11.0

**Comment by Marius1311** [comment]
Hi @rorymaizels, we have this on our radar and will get back to you later this week/early next week. Please excuse the delay!

**Comment by Marius1311** [comment]
Hi @rorymaizels, this is now implemented  (PR #512) and ready for you to use & test (thanks to @michalk8)! You will just need to install CellRank from the master branch, if you're unsure how this works, check out the "Development Version" installation instructions in our docs here: https://cellrank.readthedocs.io/en/latest/installation.html

We would appreciate to hear your feedback on this.

**Comment by rorymaizels** [comment]
Thanks guys, I'll reinstall and let you know how I get on!
